guides/common/modules/ref_system-journal-metadata.adoc

```diff
@@ -0,0 +1,51 @@
+[id="system-journal-metadata_{context}"]
+= System Journal Metadata
+
+The following table lists metadata that the `journald` service uses in {Project}.
+You can use this metadata to filter your queries.
+
+.System Journal Metadata
+[options="header"]
+|====
+|Name|Description
+|AUDIT_ACTION|Audit action performed
+
+Example: Create, update, or delete
+|AUDIT_TYPE|Audit resource type
+
+Example: Host, Subnet, or ContentView
+|AUDIT_ID|Audit resource database ID as a number
+|AUDIT_ATTRIBUTE|Audit resource field or an updated database column
+|AUDIT_FIELD_OLD|Old audit value of an update action
+|AUDIT_FIELD_NEW|New audit value of an update action
+|AUDIT_ID|Record database ID of the audit subject
+|AUDIT_ATTRIBUTE|Attribute name or column on which an action was performed
+
+Example: Name or description
+|EXCEPTION_MESSAGE|Exception message when error is logged
+|EXCEPTION_CLASS|Exception Ruby class when error is logged
+|EXCEPTION_BACKTRACE|Exception backtrace as a multiline string when error is logged
+|LOC_ID|Location database ID
+|LOC_NAME|Location name
+|LOC_LABEL|Location label
+|LOGGER
+a|Logger name
+
+To see the current list of loggers enabled by default, enter this command:
+
+[options="nowrap" subs="+quotes,attributes"]
+----
+# grep add_loggers -A14 /usr/share/foreman/config/application.rb
```

Comment:
When I run that on the latest development I see it at least misses one line, but I'd show the closing brace just to be sure:
```suggestion
# grep add_loggers -A16 /usr/share/foreman/config/application.rb
```